tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AWS/Shared/ContextPropagation.cs

```diff
@@ -24,6 +24,19 @@ private static void Inject(PropagationContext context, IDictionary messageAttrib
             sb.Append('{');
             SpanContextPropagator.Instance.Inject(context, sb, default(StringBuilderCarrierSetter));
 
+            if (dataStreamsManager != null && dataStreamsManager.IsEnabled && context.SpanContext?.PathwayContext != null)
+            {
+                var encodedBytes = PathwayContextEncoder.Encode(context.SpanContext.PathwayContext.Value);
+                var base64EncodedContext = Convert.ToBase64String(encodedBytes);
+                sb.AppendFormat("\"{0}\":\"{1}\",", DataStreamsPropagationHeaders.PropagationKeyBase64, base64EncodedContext);
+
+                if (Tracer.Instance.Settings.IsDataStreamsLegacyHeadersEnabled)
+                {
+                    // Both PropagationKeyBase64 and PropagationKey use the Base64 encoded context
+                    sb.AppendFormat("\"{0}\":\"{1}\",", DataStreamsPropagationHeaders.PropagationKey, base64EncodedContext);
```

Comment:
nit: don't use AppendFormat
```suggestion
                    ```suggestion
                sb.Append($"\"{DataStreamsPropagationHeaders.PropagationKey}\":\"");
                  .Append(base64EncodedContext)
                  .Append('"');
```
```

Reply:
makes sense, replaced.